libs/ui/elements/src/lib/thumbnail/thumbnail.component.html

```diff
@@ -1,9 +1,7 @@
 <div
   #containerElement
-  class="h-full w-full relative shrink-0 overflow-hidden flex items-center bg-gray-100"
-  [ngClass]="{
-    'bg-white': !isPlaceholder
-  }"
+  class="h-full w-full relative shrink-0 overflow-hidden flex items-center"
+  [ngClass]="isPlaceholder ? 'bg-gray-100' : 'bg-white'"
```

Comment:
Very good catch here, we basically had no guarantee that `bg-white` would have priority over `bg-gray-100` prior to this change

Reply:
The more crucial thing is actually setting the `isPlaceholder` to `false` when `updateImageList` is triggered via `ngOnChanges()` with a logo, I have the feeling. Not sure if the added tests really cover this well. But yes the html should be cleaner this way, too :-)

Reply:
yes indeed, the `bg-` classes weren't the core problem but they could have become a random and hard to track bug later on.